app/controllers/admin/base_controller.rb

```diff
@@ -0,0 +1,12 @@
+class Admin::BaseController < ApplicationController
+  before_action :require_admin
+
+  def dashboard
+    @head_title = ' | Dashboard'
```

Comment:
I would pull this out into a `DashboardHelper` instead since this is strictly for the view.